tests/helpers/test_framework_helpers.py

```diff
@@ -66,6 +67,7 @@ def test_find_suppliers_with_details_and_draft_services(mock_data_client):
         },
         (2, 'framework-slug'): {
             'frameworkInterest': {
+                'agreementId': None,
```

Comment:
how can it be countersigned but have an `agreementId` of `None`?
It's not being tested, but I would still suggest to set it to `1` so the test data is correct.